3-With-Intune MAM/IntuneClient/active-directory-xamarin-intune.Android/MainActivity.cs

```diff
@@ -17,28 +15,20 @@ public class MainActivity : global::Xamarin.Forms.Platform.Android.FormsAppCompa
     {
         private const string AndroidRedirectURI = "msauth://com.yourcompany.xamarinintuneapp/JjmT52ASxa1Lz55s+qPPgxb5xeo="; // TODO - Replace with your redirectURI
 
-        IIntuneMAMConnector _connector = new IntuneMAMConnector();
-
-        protected override void OnCreate(Bundle savedInstanceState)
+        protected override void OnMAMCreate(Bundle savedInstanceState)
         {
-            base.OnCreate(savedInstanceState);
+            base.OnMAMCreate(savedInstanceState);
 
-            Xamarin.Essentials.Platform.Init(this, savedInstanceState);
             global::Xamarin.Forms.Forms.Init(this, savedInstanceState);
 
             // configure platform specific params
             PlatformConfigImpl.Instance.RedirectUri = AndroidRedirectURI;
             PlatformConfigImpl.Instance.ParentWindow = this;
 
-            // register IntuneMAMConnector
-            Xamarin.Forms.DependencyService.RegisterSingleton<IIntuneMAMConnector>(_connector);
-
             LoadApplication(new App());
         }
-        public override void OnRequestPermissionsResult(int requestCode, string[] permissions, [GeneratedEnum] Android.Content.PM.Permission[] grantResults)
+        public override void OnRequestPermissionsResult(int requestCode, string[] permissions, [GeneratedEnum] Permission[] grantResults)
```

Comment:
This method does not seem to do anything but to call the base class. Is it worth overriding it?